src/cpp/r/session/graphics/RGraphicsDevice.hpp

```diff
@@ -46,6 +46,8 @@ int getWidth();
 int getHeight();
 double devicePixelRatio();
 
+void playDisplayList();
```

Comment:
Any thoughts on either giving this a better name, or hiding it in e.g. a `detail` namespace (since it's not something that callers would normally care about; it's just exposed so that the `rs_GEplayDisplayList` routine can call in easily)